dietpi/dietpi-cloudshell

```diff
@@ -1196,6 +1199,12 @@ _EOF_
 		# Set Nice to +10 (not critical)
 		renice -n 10 $$ &> /dev/null
 
+		# Start Tor Relay monitoring if the scene is active
+		if [[ ${aEnabledScenes[9]} == 1 ]]; then
+			nyx &
```

Comment:
Let's store the PID in a variable:
```
NYX_PID=$?
```
Then we can use `fg $NYX_PID` above and `kill -STOP $NYX_PID` below and in exit trap
```
[[ $NYX_PID ]] && kill $NYX_PID
```
in the exit trap.

This has the little bonus that in theory multiple nyx (or dietpi-cloudshell) processes can run and we are sure to only kill and control the one that was called within this instance.